packages/editor/src/hooks/useHotKeysHook.ts

```diff
@@ -0,0 +1,74 @@
+import { useHotkeys } from "react-hotkeys-hook";
+import { useDispatch } from "react-redux";
+import { setIsPanning } from "@/slices/editorSlice";
+import { AppDispatch } from "@/store";
+import useMouseKeys from "./useMouseKeys";
+
+export const useHotKeysHook = () => {
+  const dispatch = useDispatch<AppDispatch>();
+
+  useHotkeys(
+    "mousedown.2",
+    () => {
+      console.log("mousedown.2");
```

Comment:
Remove those console logs 🙂 of anything use our custom logger but it seems not usefull. All of this is in redux store

Reply:
Fixed